yaml/lib.w

```diff
@@ -0,0 +1,33 @@
+pub struct ParseOptions {
+  strict: bool?;
+}
+
+pub struct StringifyOptions {
+  indent: num?;
+}
+
+pub class Util {
```

Comment:
what's the benefit of this duplication? Why not just make the extern methods public?